docker-compose.yml

```diff
@@ -101,7 +101,7 @@ services:
       FEATURE_GET_UPDATES: 'true'
       FEATURE_ALLOW_HAVE_YOUR_SAY: 'true'
       FEATURE_PROJECT_INFORMATION: 'true'
-      PROJECT_MIGRATION_CASE_REFERENCES: 'EN010009, EN010011, EN010102, EN010085, EN010116, EN013007, EN013009, EN013012, EN013013, EN013014, EN033005, EN043005, EN043006, EN053001, TR010001, TR023018 ,TR023022, TR023024, TR033002, tr033005, TR040009, BC0110001, BC0110002'
+      PROJECT_MIGRATION_CASE_REFERENCES: 'EN010009, EN010011, EN010102, EN010085, EN010116, EN013007, EN013009, EN013012, EN013013, EN013014, EN033005, EN043005, EN043006, EN053001, TR010001, TR023018 ,TR023022, TR023024, TR033002, tr033005, TR040009, BC0110001, BC0110002, BC0110001,BC0110003,BC0110004,BC0110005'
```

Comment:
Added this to help test locally, and thought it would be useful to keep. Happy to remove if that's better.